### ISSUE TITLE
AsuraScan - bad response

### ISSUE THREAD

**Issue by marc22polo** [miru-project/repo#231]
### Extension information

AsuraScan

### Steps to reproduce

1. Open any manga from asura scans
2. Get an error: Failed to get latest data DioException
3. Click on WebView
4. Get an error: Webpage not available

### Miru version

1.8.1

### Other details

Both errors from 2. and 4. can be seen in this image
![Screenshot_20240421-131259](https://github.com/miru-project/repo/assets/26152388/a8846fb6-5ce4-4ae5-b56c-1caef8702582)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by OshekharO** [comment]
![Screenshot_20240425-094552_Miru](https://github.com/miru-project/repo/assets/95137948/32e23183-11c5-4859-b89f-bef986506979)

**Comment by marc22polo** [comment]
This issue is still not resolved, I am sorry because I failed to specify that this only happens if I try oppening manga from the home screen. When I find any manga through search, it opens fine. However when I try the same thing from home screen I get the following error.

![Screenshot_20240425-064815.jpg](https://github.com/miru-project/repo/assets/26152388/5a9bc6ed-ac2a-49a6-a1fd-45569a7710a0)

For example: I favorited The priest of corruption. If I try oppening it from the home screen it throws the original error I pasted "Failed to get latest data DioException". The same error occurs when I then try oppening any chapter from said manga. When I try to open in WebView it does not load saying web page not available ERR_NAME_NOT_RESOLVED.
But if I try to open the same manga/chapter from "search -> AsuraScans -> search for manga -> open said manga / chapter" it works completely fine.
